doc/ref/language.xml

```diff
@@ -405,7 +410,71 @@ gap> 2 * 2 + 9 = Fibonacci(7) and Fibonacci(13) in Primes;
 true
 ]]></Example>
 <P/>
-For the precedence of operators, see&nbsp;<Ref Sect="Comparisons"/>.
+
+The following table lists all operators by precedence, from highest to
+lowest, and also indicates whether the operator is left associative (aka
+left-to-right) or right associative (aka right-to-left) or neither.
+
+<Index>precedence</Index>
+<Index>associativity</Index>
+<Table Align="ll">
+<Row>
+  <Item><E>operator</E></Item>
+  <Item><E>associativity</E></Item>
+</Row>
+<HorLine/>
+<Row>
+  <Item><E>arithmetic <Ref Sect="Arithmetic Operators"/></E></Item>
+  <Item></Item>
+</Row>
+<Row>
+  <Item>&nbsp;&nbsp;&nbsp;&nbsp;<C>^</C></Item>
```

Comment:
I am abusing non breaking spaces here to force the tables to look not too bad, as a workaround for lack in GAPDoc of a way to merge table cells (something like HTML's colspan attribute for tables) 😞

Reply:
So your purpose is to achieve a kind of indentation. For those other reviewers interested in how this looks when rendered, here are the HTML and PDF versions, respectively:
<img width="784" alt="Screenshot 2021-06-29 at 09 36 48" src="https://user-images.githubusercontent.com/8956868/123765388-9a0e9e80-d8bd-11eb-9588-a5079492655c.png">
<img width="759" alt="Screenshot 2021-06-29 at 09 37 06" src="https://user-images.githubusercontent.com/8956868/123765402-9da22580-d8bd-11eb-98c8-742be7f58d1d.png">

Reply:
Thanks. I actually updated it further now:
<img width="332" alt="Screenshot 2021-07-03 at 00 30 20" src="https://user-images.githubusercontent.com/241512/124333922-db40d000-db95-11eb-8e15-3ad7038d2aeb.png">